.circleci/ci/src/pipelines/pipeline-pull-requests.ts

```diff
@@ -0,0 +1,10 @@
+import { Config } from '@circleci/circleci-config-sdk';
+import { CircleCIEnvironment } from './circleci-environment';
+import { PullRequestsWorkflow } from '../workflows';
+
+export function generatePullRequestsConfig(environment: CircleCIEnvironment): Config {
```

Comment:
Would it make sense to add a check that the branch is not `''`?

Reply:
For the moment no. Since we never trigger pipelines on tags, then branchName is never empty.